hack/verify-all.sh

```diff
@@ -73,13 +73,6 @@ if [[ "${VERIFY_GOFMT:-true}" == "true" ]]; then
   cd "${REPO_PATH}"
 fi
 
-if [[ "${VERIFY_GOLINT:-true}" == "true" ]]; then
-  echo "[*] Verifying golint..."
-  out=$(hack/verify-golint.sh 2>&1)
-  failure $? "verify-golint.sh" "${out}"
-  cd "${REPO_PATH}"
-fi
-
 if [[ "${VERIFY_GOVET:-true}" == "true" ]]; then
```

Comment:
The GOLANGCI_LINT section below should already run govet, so this can be removed (along with the `hack/verify-govet.sh` script.